src/tests/unit/mock_data/mock_recipe_provider.py

```diff
@@ -0,0 +1,80 @@
+from collections import defaultdict
+
+from domain_classes.storage_recipe import StorageAttribute, StorageRecipe
+from enums import StorageDataTypes
+
+all_storage_recipes = defaultdict(list)
+
+all_storage_recipes.update(
+    {
+        "all_contained_cases_blueprint": [
```

Comment:
Would it be a good idea to split these into files too? They aren't many, but they can be easily overlooked when they lie so far from the blueprint they relate to

Reply:
Agree. Will be done in future PR.